core-java-modules/core-java-lang-2/src/main/java/com/baeldung/pojo/EmployeeBean.java

```diff
@@ -0,0 +1,47 @@
+package com.baeldung.pojo;
+
+import java.time.LocalDate;
+
+public class EmployeeBean {
```

Comment:
In the article, you say that beans must be Serializable... this isn't - or at least it doesn't implement `Serializable`